File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/factory/transportmethod/Car.java
package com.bigboxcode.designpattern.factory.transportmethod;

public class Car implements Transport {
    @Override
    public void start() {
        System.out.println("Car started");
    }

    @Override
    public void stop() {
        System.out.println("Car Stopped");
    }

    @Override
    public void repair() {
        System.out.println("Car Repair");
    }
}

File: design-patterns/php/factory/transportmethod/Car.php
<?php
// Car.php

namespace BigBoxCode\DesignPattern\Factory\TransportMethod;

class Car implements Transport {
    public function start() {
        echo "Car started\n";
    }

    public function stop() {
        echo "Car Stopped\n";
    }

    public function repair() {
        echo "Car Repair\n";
    }
}